Sources/AppcuesKit/Presentation/Traits/TraitComposer.swift

```diff
@@ -152,3 +165,16 @@ private extension Array {
         return nil
     }
 }
+
+@available(iOS 13.0, *)
+extension Appcues {
+    func resolveEmbedViewFor(experience: Experience) -> AppcuesView? {
```

Comment:
As far as I can tell it doesn't really matter either way, so mostly just curious if there's an advantage to this being  `extension Appcues` vs a private method in `TraitComposer`?

Reply:
right - I'll move into TraitComposer for simplicity 👍